doc/user/config.toml

```diff
@@ -252,6 +252,15 @@ name = "SQL drivers"
 parent = "integrations"
 weight = 15
 
+#
+# Administration
+#
+
+[[menu.main]]
+identifier = "administration"
+name = "Administration"
+weight= 50
```

Comment:
Oh, it's just hugo frontmatter that determines the ordering in the left-hand nav bar.  So, if I had another page with weight 40, it'll be listed before this nav item.  It's unfortunate.

Reply:
Yup.  I had also considered it but wasn't quite sure because of the dbt/terraform stuff.  And, having the top level nav be collapsible as part of the IA work might be something to consider.